### DISCUSSION TITLE
graphql get middleware relationships

### DISCUSSION THREAD

**Discussion by snocksimple** [apache/skywalking#9375] [Q&A] [answered]
Hello, I'm using graphql to get the middleware call relationship in the topology, but I haven't found the right way after checking for two days, is it convenient to tell me?
For example, the application fetches the interface of mysql and redis such data?

**Comment by wu-sheng** [comment, marked as answer]
The suggestion is, to learn from UI. Any data you see on the UI could be read through GraphQL.

> For example, the application fetches the interface of mysql and redis such data?

I really can't understand what you mean.
